src/course/syllabus/application/project-gallery/learning-outcomes.md

```diff
@@ -0,0 +1,45 @@
+## HTML
+
+- [ ] Structure an HTML page using semantic tags
+- [ ] Use a variety of HTML tags to build a webpage
+- [ ] Break a page down into repeatable self-contained elements
+- [ ] Understand why we might nest HTML tags
+- [ ] Be aware of different attributes of HTML elements
+
+## CSS
+
+- [ ] Understand how to structure a stylesheet
+- [ ] Use CSS Flexbox to build a dynamic layout
+- [ ] Apply styles to elements, classes and id’s
+- [ ] Use media queries and responsive units in CSS to make the content adapt to different screen sizes
+
+## JavaScript
+
+- [ ] Understand what a Javascript function is, and how to write one
+- [ ] Reuse a function to perform repeated tasks in your code
+- [ ] Assign values to variables
+- [ ] Reassign a value to an already declared variable
+- [ ] Use loops to perform the same task on multiple elements
+- [ ] Use if statements to run code based on a condition
+
+## DOM
+
+- [ ] Select elements from the DOM and store them in variables
+- [ ] Select elements based on tag name, classes, or id's
+- [ ] Edit styles using Javascript
+- [ ] Add and remove classes from an element
+- [ ] Understand events, and how to add event listeners
+- [ ] Run a function when the user clicks a button
+
+## Asking for help
+
+- [ ] Post a code snippet on Discord
```

Comment:
I think we _could_ but it's probably unmanageable to keep track of what's repeated and where. And since they build on each other it'd be better to state it only on the earliest project.

This one was a mistake 😅